fabric-api-lookup-api-v1/README.md

```diff
@@ -0,0 +1,28 @@
+#  Fabric Api Lookup API (v1)
```

Comment:
copying my non-merge-blocking bikeshed from 1278:

I'm so sorry but I honestly think I need to commit bikeshed and say argue that "Api Lookup Api" is a terrible name that, while being very literal, is clunky to use and ends up hurting ease of understanding of the code due to the redundancy of the term "api". Calling it a component API isn't quite accurate either, since entity-component systems are inherently about managing storage as well as lookup. I think the best term to use here would be "property", as it's looking or traits of an object without caring about how they're stored.

Reply:
I think the only redundancy of the term "api" is in the name of this module, which is sometimes referred to as "API API" anyway (an API to expose APIs). In code, it only appears as `ApiLookup` or `ApiCache`.

As for the exact name, it may be a bit literal, but I think it conveys very well that this API is just meant to look up API instances from objects, I think it's more general than property.

Reply:
how about "interface lookup api" or "hook lookup api"

Reply:
I don't think those are really better. In theory, an Api doesn't necessarily have to be an interface. Also, it's mostly bikeshedding at this point... :-P